src/main/java/twofive/TwoFive.java

```diff
@@ -0,0 +1,60 @@
+package twofive;
+
+import java.io.IOException;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+
+import javafx.application.Application;
+import twofive.command.Command;
+import twofive.data.TaskList;
+import twofive.exception.TwoFiveException;
+import twofive.storage.Storage;
+import twofive.task.Task;
+import twofive.ui.Ui;
+import twofive.utils.Parser;
+
+/**
+ * Represents a chatbot that allows users to manage daily tasks.
+ */
+public class TwoFive {
+    private TaskList tasks;
+    private Storage storage;
+
+    /**
+     * Represents a constructor for the TwoFive class.
+     *
+     * @param filePath String containing file path.
+     */
+    public TwoFive(String filePath) throws IOException, TwoFiveException, DateTimeParseException {
```

Comment:
```suggestion
    public TwoFive(String filePath) throws DukeException {
```

Consider changing your exception handling to just handle DukeException which could be the umbrella exception class the other exceptions extend from

Reply:
Thank you for the suggestion and I greatly appreciate the other comments made. There is currently an umbrella exception named TwoFiveException instead of DukeException, but what you pointed out is still a good practice to keep it mind. 👍